doc/_read-me.txt

```diff
@@ -0,0 +1,14 @@
+Organization of these folders
+
+Th file  doc_xx.hs it is the configuration for java help by language.
+Th file map_xx.jhm there are the configuration of the help tree
+
+The folder icons contains the icons for doc
+
+The folder img-guide contains the pictures of the guide without translation required
+
+The folder img-libs contains the pictures of the library without translation required
+
+The folders de, el,en,fr,pt,ru are the linguistic parts of the documentation 
```

Comment:
Finish all complete sentences with with a dot ".".

Improved wording: "The folders de, el, en, fr, pt, ru contain the translated parts of the documentation in German, Spanish, English, French, Portuguese, and Russian, respectively."